README.md

```diff
@@ -38,15 +37,11 @@ To properly use Draft to containerize your application, create manifest files, a
 
 -  Supported deployment types: Helm, Kustomize, Kubernetes manifest.
 
-**Draft Set Up GitHub OpenID Connect** automates the GitHub OIDC setup process for your project.
-
-**Draft Generate Workflow** generates a GitHub Actions workflow for automatic build and deploy to a Kubernetes cluster.
-
-**Draft Add Web App Routing Annotations** annotates your Kubernetes service manifest to expose it to the internet through Web App Routing.
-
 ## Telemetry
 
-This extension collects telemetry data to help us build a better experience for building applications with Azure Kubernetes Service and VS Code.
+This extension collects telemetry data to help us build a better experience for building applications with Azure Kubernetes Service and VS Code. We only collect the following data:
```

Comment:
This is incorrect. We rely on the official `VS Code Azure SDK` tooling. It automatically tracks a few [other fields](https://github.com/Microsoft/vscode-azuretools/blob/main/utils/README.md#telemetry-and-error-handling).

Reply:
@OliverMKing added changes, want to take a look?